src/Bicep.Core/Diagnostics/DiagnosticBuilder.cs

```diff
@@ -792,7 +792,7 @@ public ErrorDiagnostic ArgumentCountMismatch(int argumentCount, int mininumArgum
             public ErrorDiagnostic LoopsNotSupported() => new(

```

Comment:
[nit] if we're calling them for-expressions in the message, then would be good to update the name of this function

Reply:
Done.